h2o-algos/src/test/java/hex/tree/TreeHandlerTest.java

```diff
@@ -571,8 +571,8 @@ public void testSlowTree() {
             assertThat("Wow, did we optimize tree rendering with language rules? It is now closer to rendering without rules!",
                     durationEnabled, greaterThan(durationDisabled * slowdownFactor));
 
-            // duration disabled and "auto" should be similar in this case, 3x factor of tolerance because we are measuring in ms
-            assertEquals(durationDisabled, durationAuto, (double) 3 * durationDisabled);
+            // duration disabled and "auto" should be similar in this case, 10x factor of tolerance because we are measuring in ms
+            assertEquals(durationDisabled, durationAuto, (double) 10 * durationDisabled);
```

Comment:
Just a though, it would be more clear to me if we hardcode it to equal +-100ms. What do you think @michalkurka